client/src/js/services/grid/GridGrouping.js

```diff
@@ -148,6 +148,13 @@ function GridGroupingService(GridAggregators, uiGridGroupingConstants, Session,
     $timeout(api.treeBase.expandAllRows, 0, false);
   }
 
+  function unfoldGroup(row) {
+    var api = this.gridApi;
+    $timeout(function () {
```

Comment:
You shouldn't need this `$timeout()` ... it's a hack and delays things getting done.  Try this:
```js
var api = this.gridApi;
api.treeBase.expandRow(row);
api.grid.notifyDataChange(uiGridConstants.dataChange.COLUMN);
```
If that works, we should also change the other `$timeout()` calls in this service.